core/src/mock_alpenglow_consensus.rs

```diff
@@ -42,6 +42,14 @@ const NUM_VOTE_ROUNDS: Slot = 4;
 /// rough upper bound of number of testnet validators to avoid allocations
 const NUM_TESTNET_VALIDATORS: usize = 1024 * 3;
 
+// Configure maximal transmission rate to be ~ 100 KPPS.
+// On average we have ~2000 destinations, so overall broadcast should
+// take at most 20 ms.
+/// Max packets to send in one batch
+const MAX_PACKETS_PER_BATCH: usize = 200;
+/// Interval between batches
+const PACING_INTERVAL: Duration = Duration::from_millis(2);
```

Comment:
Should you make this configurable?

Reply:
Should we consider latency? If the latency is higher, should we still sleep?

Reply:
I'm not sure which latency? We sleep to limit load of the switch in our own DC, latency to that is effectively zero.